docs/config.md

```diff
@@ -0,0 +1,59 @@
+## Configuring Google Spreadsheet Operations
+
+To use the Google Spreadsheet connector, add the `<googlespreadsheet.init>` element in your proxy configuration before use any other
+Google Spreadsheet connector operations. This `<googlespreadsheet.init>` element used to authenticates  the user using OAuth2 authentication
```

Comment:
The `<googlespreadsheet.init>` element authenticates the user using OAuth2 authentication and it allows users to access the Google account which contains the spreadsheets.